src/Altinn.App.Core/Internal/App/AppFeatures.cs

```diff
@@ -0,0 +1,24 @@
+namespace Altinn.App.Core.Internal.App
+{
+    /// <summary>
+    /// Default implementation of IAppFeatures
+    /// </summary>
+    public class AppFeatures : IAppFeatures
```

Comment:
We should perhaps document the difference between this feature handling and the FeatureManagement use somewhere in the code as it's there it might be confusing for others (and future me).

Reply:
Should we consider renaming our implementation to capabilities to make the likelihood of confusing it with FeatureManagement?
@olemartinorg if we decide to rename this, does it seem like a better name in appmetadata? we are reporting if the backend has the capability X not a feature?

Reply:
Fine by me - but we'd need to change it in the next release of app-frontend in that case (the multipart-save feature/capability is already in production).

Let's consider the two cases we have now:
- Footer. This is a new capability the backend magically gets in a new version.
- Multipart saving (should probably be named "changelist in data write" or something on the backend), which is a capability you should be able to turn on/off in apps (and which is on by default for new apps, and will be the only option in some future major version).

From that, I think 'features' sound better, because features can be toggled and capabilities sounds like they cannot be toggled. The difference is that `FeatureManagement` lets an app developer control the backend/frontend features of their app, while `AppFeatures` lets the backend toggle features in app-frontend. So maybe `FrontendFeatures` is a better name? 🤷

Reply:
Agreed, was only thinking of footer when I came up with capabilities. FrontendFeatures sound better as our intent is to report what frontend features the backend supports and has enabled. Any objections to FronendFeatures @RonnyB71 ?

Reply:
Renamed it to FrontendFeatures